### PR TITLE
[TEIID-5570] Upgrading apache poi to 3.17

### PR DESCRIPTION
**Pull request by vhalbert**

### CODE REVIEW

File: connectors/file/translator-excel/src/main/java/org/teiid/translator/excel/ExcelExecution.java [hunk 1]
```diff
@@ -73,7 +73,7 @@ public ExcelExecution(Select query, ExecutionContext executionContext,
                 continue;
             }
 
-            Cell cell = row.getCell(index-1, Row.RETURN_BLANK_AS_NULL);
+            Cell cell = row.getCell(index-1, Row.MissingCellPolicy.CREATE_NULL_AS_BLANK);
```
Comment (shawkins) [inline connectors/file/translator-excel/src/main/java/org/teiid/translator/excel/ExcelExecution.java]: It looks like this should be Row.MissingCellPolicy.RETURN_BLANK_AS_NULL instead

**Comment by shawkins** [comment]
Also https://issues.jboss.org/browse/TEIID-5570 doesn't seem to be visible.  Is that the correct jira?

**Comment by vhalbert** [comment]
Not sure what's happening on accessing this jira.  I have it opened and can see it, but if I use this link that is sent in the email, it won't let me view it.  Not sure what is happening yet.

**Comment by vhalbert** [comment]
Corrected the reference

**Pull request merged**